File: buscaSequencial/BuscaSequencial.kt
fun main() {
    val array = intArrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
    println(buscaSequencial(array, 8))
}

fun buscaSequencial(array: IntArray, valor: Int): Int {
    for (i in array.indices){
        if (array[i] == valor){
            return i
        }
    }
    return -1
}

File: buscaSequencial/buscaSequencial.js
function busca_sequencial(array, valor) {
    for (let i = 0; i <= array.length - 1; i++) {
        if (array[i] == valor){
            return i;
        }
    }
    return -1
}

const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
console.log(busca_sequencial(array, 8))